src/betterproto/plugin/typing_compiler.py

```diff
@@ -139,28 +139,32 @@ def imports(self) -> Dict[str, Optional[Set[str]]]:
 class NoTyping310TypingCompiler(TypingCompiler):
```

Comment:
i tried my hand at fixing the tests that broke in `test_typing_compiler.py`:
```
def test_no_typing_311_typing_compiler():
    compiler = NoTyping310TypingCompiler()
    assert compiler.imports() == {}
    assert compiler.optional("str") == '"str | None"'
    assert compiler.imports() == {}
    assert compiler.list("str") == '"list[str]"'
    assert compiler.imports() == {}
    assert compiler.dict("str", "int") == '"dict[str, int]"'
    assert compiler.imports() == {}
    assert compiler.union("str", "int") == '"str | int"'
    assert compiler.imports() == {}
    assert compiler.iterable("str") == "Iterable[str]"
    assert compiler.imports() == {"collections.abc": {"Iterable"}}
    assert compiler.async_iterable("str") == "AsyncIterable[str]"
    assert compiler.imports() == {"collections.abc": {"Iterable", "AsyncIterable"}}
    assert compiler.async_iterator("str") == "AsyncIterator[str]"
    assert compiler.imports() == {
        "collections.abc": {"Iterable", "AsyncIterable", "AsyncIterator"}
    }
```

these pass the tests when running `poe test`